projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/use-protect-tooltip-copy.ts

```diff
@@ -13,24 +20,136 @@ export type TooltipContent = {
 /**
  * Gets the translated tooltip copy based on Protect Scan details.
  *
- * @param {object} props - React props
- * @param {number} props.pluginsCount - The number of installed plugins on the site.
- * @param {number} props.themesCount - The number of installed themes on the site.
- * @param {number} props.numThreats - The number of detected scan threats on the site.
  * @returns {TooltipContent} An object containing each tooltip's title and text content.
  */
-export function useProtectTooltipCopy( {
-	pluginsCount = 0,
-	themesCount = 0,
-	numThreats = 0,
-}: {
-	pluginsCount?: number;
-	themesCount?: number;
-	numThreats?: number;
-} = {} ): TooltipContent {
+export function useProtectTooltipCopy(): TooltipContent {
 	const slug = 'protect';
 	const { detail } = useProduct( slug );
-	const { hasPaidPlanForProduct: hasProtectPaidPlan } = detail;
+	const { isPluginActive: isProtectPluginActive, hasPaidPlanForProduct: hasProtectPaidPlan } =
+		detail;
```

Comment:
```suggestion
		detail || {};
```

Probably have this here just to be safe 😄. Also should probably have this for the `scanData` and `wafData` below

Reply:
Thanks!  Yes, I handled every occurrence of this when destructuring myJetpackInitialState, throughout all these Protect card files. 👍